### PR TITLE
Fixes #5379 No verification email

### LINKED ISSUE #5379: No verification email
**Describe the bug**
No verification email is being sent 
**To Reproduce**
Sign up and you won't receive a verification mail

**Expected behavior**
You should receive a verification email

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #5379 No verification email

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

Fixes #5379 No verification email

#### Changes proposed in this pull request:

- Fixes #5379 No verification email

Proof : 

email-verify

@fossasia/open-event-server-devs Please review !

### CODE REVIEW

File: app/api/helpers/mail.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def send_email(to, action, subject, html):
                     "Content-Type": "application/json"
                 }
                 from .tasks import send_email_task
-                send_email_task.delay(payload, headers)
+                send_email_task(payload, headers)
```
Comment (iamareebjamal) [inline app/api/helpers/mail.py]: Will this change sending the email from asynchronously to synchronously?

Comment (bhaveshAn) [reply app/api/helpers/mail.py]: Yes it sends the mail immediately. 
.delay wasn't working actually.

Comment (realslimshanky) [reply app/api/helpers/mail.py]: Can we first debug the `delay` method? It would be better to keep this feature async.

Comment (iamareebjamal) [reply app/api/helpers/mail.py]: That's what I meant. This is a workaround, not a fix

Comment (bhaveshAn) [reply app/api/helpers/mail.py]: Fair point. But for now we want to make it run as less than a week left for jugaadfest. No events canbe published due to this issue.

Comment (iamareebjamal) [reply app/api/helpers/mail.py]: Add a TODO comment and open a new issue regarding fixing this

**Comment by bhaveshAn** [comment]
@iamareebjamal Please merge !

**Pull request merged**